agent/agent.go

```diff
@@ -75,9 +77,22 @@ func (a *Agent) Commit() (err error) {
 }
 
 func (a *Agent) Push() error {
-	return a.repo.Push(&git.PushOptions{
-		Force: a.Config.Push.Force,
-	})
+	if err := a.repo.Push(&git.PushOptions{}); err != nil && err != git.NoErrAlreadyUpToDate {
```

Comment:
If I understand correctly, this bit does the normal push and the following does --tags push. If that is the case, shouldn't force push apply here too?

Reply:
yikes, good catch